connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -453,11 +466,12 @@ class SparkSession private[sql] (
     client.execute(plan).asScala.foreach(_ => ())
   }
 
-  private[sql] def execute(command: proto.Command): Unit = {
+  private[sql] def execute(command: proto.Command): Seq[ExecutePlanResponse] = {
```

Comment:
Why is this change needed?

Reply:
This can be used for RPCs where response is required (RemoteStreamingQuery methods use this).
Alternately we can add another method. But does not seem necessary. Better for caller to explicitly ignore them.